backend/src/main/java/org/cryptomator/hub/entities/UnlockResult.java

```diff
@@ -0,0 +1,7 @@
+package org.cryptomator.hub.entities;
+
+public enum UnlockResult {
+	SUCCESS,
+	NO_SUCH_DEVICE,
+	DEVICE_NOT_AUTHORIZED
+}
```

Comment:
@overheadhunter Why is this enum standalone?

Also, similar to slf4j and its log levels, we can even add for each entry an nice, static convenience factory method:
```java
public static logSuccess(...) {
    log(..., SUCCESS);
}

public static logNoSuchDevice(...) {
    log(..., NO_SUCH_DEVICe);
}
...
```

Reply:
I read your comment and fixed it in https://github.com/cryptomator/hub/pull/192/commits/4532f8badd8bb3f5459cdb67dca7b83a6f46defd